gpgme.go

```diff
@@ -546,6 +548,18 @@ func (c *Context) Encrypt(recipients []*Key, flags EncryptFlag, plaintext, ciphe
 	return handleError(err)
 }
 
+func (c *Context) SetPassphrase(passphrase string) {
+	logrus.Debugf("Setting GPGME passphrase callback", passphrase)
+	callback := C.gpgme_passphrase_cb_t(C.passphrase_cb)
+	cPass := C.CString(passphrase)
+	C.gpgme_set_pinentry_mode(c.ctx, C.GPGME_PINENTRY_MODE_LOOPBACK);
+	C.gpgme_set_passphrase_cb(c.ctx, callback, unsafe.Pointer(cPass))
```

Comment:
These are all unsafe WRT CGo rules, per https://github.com/proglottis/gpgme/pull/23 .

There already is a `Context.SetCallback` exposing `gpgme_set_passphrase_cb` (and see how different that implementation needs to be), isn’t that sufficient already?

To the extent actually using `SetCallback` with a fixed string requires some code that could be widely reused, it might well make sense to provide that in this `gpgme` package, but I think it would be cleaner to just wrap `SetCallback` instead of reimplementing it, and it’s up to the upstream maintainer whether to add it.